vnet/spec/vnet/endpoints/1.0/networks_spec.rb

```diff
@@ -21,24 +21,31 @@ def app
   include_examples 'DELETE /:uuid'
 
   describe 'POST /' do
+    let!(:topology) { Fabricate(:topology) { uuid 'topo-test' }  }
+
     accepted_params = {
-      :uuid => 'nw-test',
-      :display_name => 'our test network',
-      :ipv4_network => '192.168.2.0',
-      :ipv4_prefix => 24,
-      :domain_name => 'vdc.test.domain',
-      :network_mode => 'virtual',
+      uuid: 'nw-test',
+      display_name: 'our test network',
+      ipv4_network: '192.168.2.0',
+      ipv4_prefix: 24,
+      domain_name: 'vdc.test.domain',
+      network_mode: 'virtual',
+      topology_uuid: 'topo-test'
+    }
+    expected_response = accepted_params.dup.tap { |map|
+      map.delete(:topology_uuid)
```

Comment:
This way we don't test if topology uuid was set as expected. I think either toplogy_uuid should be included in some form in the response or we should add a test here (and also for route links and segments) that checks the database to see if the correct relationships were made.